src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString.split(';');
+  const clearStyles = styles
+    .map((style) => style.trim())
+    .filter((style) => style.length > 0);
+  const result = clearStyles.reduce((prev, style) => {
+    const property = style.split(':')[0].trim();
```

Comment:
You are splitting the 'style' string twice, once for the property and once for the value. This is redundant. You can split it once and use destructuring to get both parts, which is more efficient.